### DISCUSSION TITLE
Support Request - API Login 404

### DISCUSSION THREAD

**Discussion by Diogo96ferreira** [speechly/speechly#218] [Q&A]
Hello everyone, 

I'm currently using Speechly to create an Expense Tracker App, following a [Youtube Video](https://www.youtube.com/watch?v=NnUFOWR_V4Y). By the time i reach the integration with Speechly, i'm receiving a 404 error when the API call is made. 

Already created 2 different Speechly Applications, in order to validate if there was any error on the first i created, but it persists. Made sure i got the AppID and the language properly set up, and tried it on Codepen with my AppID, and it works. whenever i try to use it on my side, it breaks. 

May it be something regarding the Versions that i'm currently using? I'll leave the package.json below so that option can be validated.
I'm using Node.js v7.0.7 

Thank you,
Diogo

`{
	"name": "speechly",
	"version": "0.1.0",
	"private": true,
	"dependencies": {
		"@material-ui/core": "^4.12.4",
		"@material-ui/icons": "^4.11.3",
		"@material-ui/lab": "^4.0.0-alpha.61",
		"@speechly/react-client": "^0.0.4",
		"@speechly/react-ui": "^2.7.0",
		"@speechly/speech-recognition-polyfill": "^1.2.0",
		"@testing-library/jest-dom": "^5.11.6",
		"@testing-library/react": "^11.2.2",
		"@testing-library/user-event": "^12.5.0",
		"chart.js": "^2.9.4",
		"react": "^16.12.0",
		"react-chartjs-2": "^2.11.2",
		"react-dom": "^16.12.0",
		"react-scripts": "^2.1.3",
		"uuid": "^8.3.2",
		"web-vitals": "^0.2.4"
	},
	"scripts": {
		"start": "react-scripts start",
		"build": "react-scripts build",
		"test": "react-scripts test",
		"eject": "react-scripts eject"
	},
	"browserslist": {
		"production": [
			">0.2%",
			"not dead",
			"not op_mini all"
		],
		"development": [
			"last 1 chrome version",
			"last 1 firefox version",
			"last 1 safari version"
		]
	}
}
`

**Comment by langma** [comment]
Hi Diogo, 

The `package.json` you attached is not installable for me. Try removing the `@speechly/react-client` dependency, as that is installed along with the react-ui package.

Are you sure you are using node `v7.0.7`? I'd say it would be safest to use the current LTS release, which is `v16.16.0` (https://nodejs.org/en/download/).

**Reply by Diogo96ferreira** [reply]
Hello @langma . 

Trying those changes atm, will reply onec i got them done with feedback. Thanks for the help

**Reply by Diogo96ferreira** [reply]
Hello @langma  
Seems like it actually needs the react-client to compile properly. 
I've installed the new node.js version, and now im having a compilation error on my push to talk button regarding undefined properties. i'll leave the error and how i iimplemented it bellow: 

Error: 
Uncaught TypeError: Cannot read properties of undefined (reading 'Idle')
The above error occurred in the <PushToTalkButton> component:

Implementation: 
`
import { PushToTalkButton } from "@speechly/react-ui";

<PushToTalkButton
				placement="bottom"
				hide="false"
				captureKey=" "
				intro="Push to talk"
				size="80px"
			/>`

The error is hitting on the PushToTalk.tsx:54

thanks once again
